include/picongpu/plugins/common/DumpTimes.hpp

```diff
@@ -0,0 +1,145 @@
+#include <chrono>
+#include <cstdlib>
+#include <ctime>
+#include <fstream>
+#include <functional>
+#include <iomanip>
+#include <sstream>
+#include <string>
+#include <utility>
+
+namespace picongpu
+{
+    namespace openPMD
+    {
+        template<typename Clock>
+        struct TimeFormatters
+        {
+            using TimeFormatter = std::function<std::string(typename Clock::time_point const&)>;
+
+            static std::string humanReadable(typename Clock::time_point const& currentTime)
+            {
+                std::stringstream res;
+                std::time_t now_c = Clock::to_time_t(currentTime);
+                res << std::put_time(std::localtime(&now_c), "%F %T") << '.' << std::setfill('0') << std::setw(3)
+                    << std::chrono::duration_cast<std::chrono::milliseconds>(currentTime.time_since_epoch()).count()
+                        % 1000;
+                return res.str();
+            }
+
+            static std::string epochTime(typename Clock::time_point const& currentTime)
+            {
+                return std::to_string(
+                    std::chrono::duration_cast<std::chrono::milliseconds>(currentTime.time_since_epoch()).count());
+            }
+        };
+        template<typename Clock>
```

Comment:
` = std::chrono::milliseconds` then we can remove it from a bunch of code but it still allows us to change the precision.